pkgs/shelf_router/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.6
+
+* Added function `getParams` to the `Router` class to get the parameters from a request
+
 ## 1.1.5-wip
```

Comment:
```suggestion
## 1.2.0-wip

* Added function `getParams` to the `Router` class to get the parameters from a request
```

Reply:
merge the `1.1.5-wip` bits into something like

```markdown
## 1.2.0-wip

* Added function `getParams` to the `Router` class to get the parameters from a request
* Require Dart `^3.3.0`.
```

and update the pubspec accordingly!

Reply:
Since this adds a feature

Reply:
Alright, looks good to me!